docs/testing.md

```diff
@@ -7,6 +7,10 @@ Foreman Ansible Modules are tested in different ways:
 2. Integration tests
 3. Ansible Sanity
 
+Before you start, ensure all required dependencies are installed. Typically this is `python, python-devel, popt-devel`. To create a virtual environment, do:
+
+    python3 -m venv venv --system-site-packages
+
```

Comment:
this *shouldn't* be necessary, if it was for you, we shall debug that together, outside this PR

Reply:
Removing the hunk then.